monai/utils/img2tensorboardutils.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import PIL
+from tensorboard.compat.proto import summary_pb2
+
+
+def _image3_animated_gif(imp, scale_factor=1):
+    """
+    Function to actually create the animated gif.
+    Args:
+        imp: tuple of tag and a list of image tensors
+        scale_factor: amount to multiply values by (if the image data is between 0 and 1, using 255 for this value will
+        scale it to displayable range)
+    """
+    from PIL.GifImagePlugin import Image as GIF
```

Comment:
Suggest to move "import" at the beginning.

Reply:
Done